force_bdss/mco/optimizer_engines/aposteriori_optimizer_engine.py

```diff
@@ -29,19 +29,23 @@ class AposterioriOptimizerEngine(BaseOptimizerEngine):
     #: callable
     optimizer = Instance(IOptimizer, transient=True)
 
-    def optimize(self, *vargs):
+    def optimize(self, **kwargs):
         """ Generates optimization results.
 
```

Comment:
It might help to have docstrings for the **kwargs.

Reply:
Has been updated